backend/backend.containerfile

```diff
@@ -18,6 +18,7 @@ EXPOSE 8000
 RUN dnf install -y pip gcc python3-devel gcc-c++
 
 RUN pip install --user poetry && \
+    poetry self add poetry-plugin-export && \
```

Comment:
Needs this fix because of upstream [centos:stream9](https://quay.io/repository/centos/centos?tab=tags&tag=latest) changes (i.e has Poetry 2.0 in its release repository) 4 days ago.

Reply:
This is interesting, as it should affect any containerized build, which I'd guess Varshini will need on her branch.

Reply:
Yes. cc: @MVarshini